src/docs/content/disaster-recovery/setup.md

```diff
@@ -0,0 +1,273 @@
+---
+title: "Setup"
+date: 2021-08-10T13:29:23-06:00
+draft: false
+weight: 3
+tags:
+  - Autonomous Data Guard
+  - Autonomous Transaction Processing
+  - Disaster Recovery
+---
+
+### Introduction
+
+In this section we detail the steps required to perform a DR solution between ap-mumbai-1 and ap-hyderabad-1 regions. These regions are used as an example, you can choose the regions of your choice.
+
+### Set Environment Variables
+   
+    export COMPARTMENT_ID=Your_compartment_id
+    export DB_NAME=demoadb
+    export DB_DISPLAY_NAME=demoadb
+    export DB_PASSWORD=DemoAdb#1234
```

Comment:
Please, do not use literals for the variables examples, and avoid at all cost for passwords.

On the Lines 88-93 you correctly used the <VariableNote>